File: storydoc-frontend/src/app/api/models/story-doc-dto.ts
/* tslint:disable */
/* eslint-disable */
import { BlockDto } from './block-dto';
import { ResourceUrn } from './resource-urn';
import { StoryDocId } from './story-doc-id';
export interface StoryDocDto {
  blocks?: Array<BlockDto>;
  storyDocId?: StoryDocId;
  title?: string;
  urn?: ResourceUrn;
}

File: storydoc-backend/src/main/java/io/storydoc/server/storydoc/app/dto/StoryDocDTO.java
package io.storydoc.server.storydoc.app.dto;

import io.storydoc.server.storydoc.domain.StoryDocId;
import io.storydoc.server.workspace.domain.ResourceUrn;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class StoryDocDTO {

    private StoryDocId storyDocId;

    private List<BlockDTO> blocks;

    private String title;

    private ResourceUrn urn;
}
